src/main/resources/cli-help/ballerina-dist-remove.help

```diff
@@ -2,17 +2,23 @@ NAME
        ballerina-remove - Remove distributions in your local environment
 
 SYNOPSIS
+       ballerina dist remove <-a | --all>
        ballerina dist remove <distribution>
 
 DESCRIPTION
-       Remove deletes the given Ballerina distribution from your local environment.
+       Remove deletes the given Ballerina distribution from your local environment if you specify the version.
 
        However, you are not allowed to remove the active distribution.
 
        Here, the "distribution" has two parts: name and version. Following are some sample
-       distributions: jballerina-1.1.0, jballerina-1.4.0, nballerina-0.5.0.
+              distributions: jballerina-1.1.0, jballerina-1.4.0, nballerina-0.5.0.
+
+       Remove deletes the all Ballerina distribution except currently active distribution from your local
+       environment if you issue <-a | --all> flag.
```

Comment:
The <-a | --all> flag deletes all the Ballerina distributions except the currently active one from your local
       environment.